src/ipc.cpp

```diff
@@ -291,9 +262,19 @@ void IPC::processEvents()
             auto it = eventHandlers.find(name);
             if (it != eventHandlers.end())
             {
-                evt->accepted = runEventHandler(it.value(), evt->data);
                 qDebug() << "IPC: Processing event: " << name << ":" << evt->posted << "=" << evt->accepted;
```

Comment:
When new event struct is allocated its zeroed [here](https://github.com/novist/qTox/blob/ipc-improvements/src/ipc.cpp#L116) so its ok, its still initialized to false.